### PR TITLE
[HAC-163] As a developer I have header without grommet

### PR DESCRIPTION
**Pull request by kkotelczuk**
- Remove grommet completely from project
- Change Header style

### CODE REVIEW

**Comment by kkkowalski** [comment]
LGTM

**Comment by StudentDawid** [comment]
LGTM

File: src/apps/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Logo from '../../common/Logo';
+import Menu from '../Menu';
+import Radium from 'radium';
+
+export default Radium(() => {
+  const styles = {
+    headerBackground: {
+      backgroundColor: '#FFFFFF',
+      margin: '0 -9999rem',
```
Comment (patrykkopycinski) [inline src/apps/Header/Header.jsx:10]: why?

Comment (kkotelczuk) [inline src/apps/Header/Header.jsx:10]: It is used to expand header background to screen width

**Comment by patrykkopycinski** [comment]
lgtm

File: src/apps/Header/Header.jsx (hunk 1 above)
Comment (patrykkopycinski) [inline src/apps/Header/Header.jsx:10]: ok, hackish, but fine ;)

**Pull request merged**